api/src/main/resources/applicationContext-service.xml

```diff
@@ -680,6 +477,7 @@
 	
 	<cache:annotation-driven cache-manager="apiCacheManager"/>
 	
-	<tx:annotation-driven />
+	<tx:annotation-driven transaction-manager="transactionManager"/>
+	<aop:aspectj-autoproxy />
```

Comment:
I don't think we need aspectj-autoproxy if we use the AOPConfig from above. Spring AOP should be good enough for us.